source/docs/advanced/performance/preloading-routes.md

```diff
@@ -0,0 +1,183 @@
+---
+id: preloading-routes
+title: Preloading Routes' data
+---
+
+When a website is slow, the first thing that comes to mind is to improve raw metrics like the Speed Index. And while this is definitly useful, we shouldn't overlook the [_perceived_ performances](https://blog.teamtreehouse.com/perceived-performance). The goal is not to improve the response time, but to better manage the time to make the user _think_ that the website is fast.
+
+One way to do this is to anticipate the user's need and start the long running tasks before they actually need it. For instance, when a user is navigating on a catalog, on desktop, they will hover the product before clicking on it. Thus, we can trigger the product's load as soon as they hover the product, instead of waiting for the click. This can seem like small gains, but this definitly improves how fast the user _thinks_ your site is.
```

Comment:
```suggestion
One way to do this is to anticipate the user's need and start the long running tasks before they actually need it. For instance, when a user is navigating on a catalog, on desktop, they will hover the product before clicking on it. Thus, we can start loading the product page as soon as the user hover the product element, instead of waiting for the click. This can seem like a micro-optimization, but it definitely improves how fast the user _thinks_ your site is.
```